modules/core/shared/src/main/scala/lucuma/core/optics/SplitMono.scala

```diff
@@ -36,15 +36,15 @@ final case class SplitMono[A, B](get: A => B, reverseGet: B => A) {
     SplitEpi(reverseGet, get)
 
   /** Compose with another SplitMono. */
-  def composeSplitMono[C](f: SplitMono[B, C]): SplitMono[A, C] =
+  def andThen[C](f: SplitMono[B, C]): SplitMono[A, C] =
     SplitMono(get.andThen(f.get), reverseGet.compose(f.reverseGet))
 
   /** Compose with another SplitEpi. */
-  def composeSplitEpi[C](f: SplitEpi[B, C]): Wedge[A, C] =
+  def andThen[C](f: SplitEpi[B, C]): Wedge[A, C] =
     Wedge(get.andThen(f.get), reverseGet.compose(f.reverseGet))
 
   /** Compose with an Iso. */
-  def composeIso[C](f: Iso[B, C]): SplitMono[A, C] =
+  def andTen[C](f: Iso[B, C]): SplitMono[A, C] =
```

Comment:
Typo here. I guess we don't have a test for this ;-)

Reply:
Not a typo, just spoken with an accent.